Cython/Includes/cpython/datetime.pxd

```diff
@@ -210,3 +210,12 @@ cdef inline int timedelta_seconds(object o):
 # Get microseconds of timedelta
 cdef inline int timedelta_microseconds(object o):
     return (<PyDateTime_Delta*>o).microseconds
+
+cdef inline double total_seconds(timedelta obj):
+    # Mirrors the timedelta.total_seconds method
```

Comment:
I think we should still add a comment that this uses more floating point operations and thus is not guaranteed to give the same result. (It probably does for reasonable time ranges, though.)
```suggestion
    # Mirrors the "timedelta.total_seconds()" method.
    # Note that this implementation is not guaranteed to give *exactly* the same
    # result as the original method, due to potential differences in floating point rounding.
```